src/app/private/board/page.tsx

```diff
@@ -0,0 +1,22 @@
+import Header from "@components/Header";
+import { Button, Typography } from "@mui/material";
+
+const Page = () => {
+  return (
+    <div>
+      <Header
```

Comment:
I think it might be better to put Header components into layout.tsx

Reply:
oh wait unless we don't want it to display for nested pages/layouts, i just re-checked the figma

Reply:
Yeah I couldn't think of an elegant solution for a Layout component because each Tab (profile, board, queue, profile) and their nested pages have different headers

Reply:
yeah makes sense i think it looks good for now 👍